src/components/calculator/ExplanationCard/ExplanationCard.tsx

```diff
@@ -25,6 +25,7 @@ import {
   calculatePersonRiskEach,
 } from 'data/calculate'
 import { Interaction, budgetOptions, intimateDurationFloor } from 'data/data'
+import './ExplanationCard.scss'
```

Comment:
Hm can we do this without relative imports? It's much harder to refactor as is. See data/data and how it can bw imported from the project root in the same file right about this.

Reply:
Okay done!